superset-frontend/src/components/Select/Select.tsx

```diff
@@ -77,12 +74,6 @@ export type OptionsTypePage = {
   totalCount: number;
 };
 
-export type OptionsPagePromise = (
-  search: string,
-  page: number,
-  pageSize: number,
-) => Promise<OptionsTypePage>;
-
 export type SelectRef = HTMLInputElement & { clearCache: () => void };
```

Comment:
We should remove the `SelectRef` as well because there's no more cache. Just use `HTMLInputElement` as the ref type.

We should also remove `clearCache` and `useImperativeHandle` below.